docs/modules/ROOT/pages/gsn.adoc

```diff
@@ -0,0 +1,77 @@
+= The Gas Station Network
+
+User onboarding is one of the hottest topics in Ethereum, and meta-transactions the de-facto solution to this important problem. Here you'll learn all about the Gas Station Network (GSN) and how to write contracts that don't require their users to hold Ether to pay for gas.
+
+If you're already up to speed with the workings of the GSN, feel free to skip to [receiving-a-relayed-call], where we'll go over how to use the OpenZeppelin Contracts to easily write a GSNRecipient contract. Otherwise, strap in!
+
+== Sending gas-less transactions
+
+All Ethereum transactions spend gas, and the sender of the transaction must have enough Ether to pay for these gas costs. Even though these costs are low for basic transactions (a couple cents), getting this Ether is no easy task: you'll probably need to go through Know Your Customer and Anti Money-Laundering processes (KYC & AML), which not only take time but often involve sending a selfie holding your passport over the Internet (!). Only the most hardcore users will put up with this hassle, and dApp adoption greatly suffers when Ether is required. We can do better.
+
+**Enter meta-transactions**. This is a fancy name for a simple idea: a third-party can send another user's transactions and pay themselves for the gas cost. That's it! There's some tricky technical details, but those can be safely ignored when interacting with the GSN. This way, instead of your users calling into your contract (called the _recipient_) directly, someone else (we'll call them a _relay_) will send their transaction and pay for the cost.
```

Comment:
Are we using "relay" or "relayer"?

Reply:
Also, let's clarify the reader that they don't need to run a relayer themselves: they are run by other parties in the network. This is one of the key differences with other meta-tx solutions.

Reply:
I worry that may confuse a majority of users not familiar with the details behind other solutions (for one, it would confuse me). Perhaps the distinction you mentioned is better suited to a high-level explanation of the GSN that compares it to other systems?

The following sentences immediately refer to the relayer's incentives, why they are paid, trusting them, etc., and generally speak of 'they', which I think is clear enough for this context.